### PR TITLE
Reece 06 correct answer display win to Dev

### PR DESCRIPTION
**Pull request by RCollettSG**
Hi Richard/Anthony, Please could you review my pull request

Thank you

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -19,23 +18,95 @@ $(function() {
   //get questions from an API
   function getQuestions(difficulty) {
     switch (difficulty) {
-
       case "easy":
-        $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple", function(data){
-          return data;
+          $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=easy&type=multiple", function(data){
+          questions = data;
```
Comment (AEJackson) [inline js/main.js]: check indentation

File: js/main.js [hunk 2]
```diff
@@ -19,23 +18,95 @@ $(function() {
...
+          return questions;
         })
         break;
       case "medium":
-        $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=medium&type=multiple", function(data){
-          return data;
-
+          $.get("https://opentdb.com/api.php?amount=50&category=9&difficulty=medium&type=multiple", function(data){
+          questions = data;
```
Comment (AEJackson) [inline js/main.js]: check indentation

File: js/main.js [hunk 3]
```diff
@@ -19,23 +18,95 @@ $(function() {
...
+    var randQues = Math.floor(Math.random() * 50);
+    var randBtn = Math.floor(Math.random() * 4) + 1;
+    //Display the first question
+    $("#question").html(ques.results[randQues].question);
+    //Display the correct answer on a random button.
+    $("#answer" + randBtn).html(ques.results[randQues].correct_answer).addClass("correct")
+    //Display the incorrect answers in the remaining buttons
+    for (var i = 0; i < answerBtns.length; i++) {
+      if(!$(answerBtns[i]).hasClass("correct")) {
+        debugger;
```
Comment (AEJackson) [inline js/main.js]: remove debugger;

**Comment by RCollettSG** [comment]
Hi Anthony, I have made the changed you have requested

Thank you

**Pull request merged**